sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/ChangeFeedProcessorItem.java

```diff
@@ -17,6 +19,7 @@
  */
 @Beta(value = Beta.SinceVersion.V4_37_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
 public final class ChangeFeedProcessorItem {
+    private final static ObjectMapper mapper = Utils.getSimpleObjectMapper();
```

Comment:
We should not store this, as it just returns a reference, so we can call it as many times. 
Lets not make a new reference.